prototype/frameworks/llamastack/scripts/rag.py

```diff
@@ -1,137 +1,105 @@
 import os
-import uuid
 from pathlib import Path
 from termcolor import cprint
 from dotenv import load_dotenv
-from llama_stack_client.lib.agents.agent import Agent
-from llama_stack_client.lib.agents.event_logger import EventLogger
-from llama_stack_client.types.agent_create_params import AgentConfig
-from llama_stack_client.types import Document
+from llama_stack_client import LlamaStackClient, Agent
+from llama_stack_client import AgentEventLogger
+from llama_stack_client import RAGDocument
 
 load_dotenv()
 
 # Configuration
-LLAMA_STACK_PORT = os.environ.get("LLAMA_STACK_PORT", "80")
-INFERENCE_MODEL = os.environ.get("INFERENCE_MODEL", "meta-llama/Llama-3.2-3B-Instruct")
-
-FILE_PATHS = ["documents/cake.rst", 
-              "documents/car.rst", 
-              "documents/house.rst", 
-              "documents/technology.rst"]
-VECTOR_DB_PROVIDER = "faiss"
-EMBEDDING_MODEL = "all-MiniLM-L6-v2"
-EMBEDDING_DIMENSION = 384
-CHUNK_SIZE = 512
-TOP_K = 3
-MAX_TOKENS_IN_CONTEXT = 4096
-MAX_CHUNKS = 2
-
+LLAMA_STACK_PORT = os.getenv("LLAMA_STACK_PORT")
+INFERENCE_MODEL = os.getenv("INFERENCE_MODEL")
 
 def create_http_client():
     """Creates an HTTP client for communicating with the Llama Stack server."""
-    from llama_stack_client import LlamaStackClient
     endpoint = os.getenv("LLAMA_STACK_ENDPOINT")
     base_url = endpoint if endpoint else f"http://localhost:{LLAMA_STACK_PORT}"
     return LlamaStackClient(
         base_url=base_url
     )
 
-
 def create_library_client(template="ollama"):
     from llama_stack import LlamaStackAsLibraryClient
     client = LlamaStackAsLibraryClient(template)
     client.initialize()
     return client
 
-
 client = create_http_client()  # or create_http_client() depending on the environment
 
-# Load documents for RAG
+# there are few documents in the `llamastack/documents` folder, make sure to update the file path below with your local directory path
+FILE_PATHS = ["<your local path for the repo dir>/documents/cake.rst", 
+              "<your local path for the repo dir>/documents/car.rst", 
+              "<your local path for the repo dir>/documents/house.rst", 
+              "<your local path for the repo dir>/documents/technology.rst"]
+
```

Comment:
to make it possible to run as/is, please can you update the paths to "documents/.." as you had before? The comment you added above will point users in the right direction. I think your original change of assuming running from the same directory as the documents folder makes sense, it only tripped me up bc before I found I needed to back up a directory to run the script. Now I don't!